Week3/homework/step3.js

```diff
@@ -1,8 +1,9 @@
 'use strict';
 
 function createBase(base) {
-  // Replace this comment and the next line with your code
-  console.log(base);
+  return function(num) {
+    return num + base;
```

Comment:
Nice! You could even try using an arrow function here, which would reduce the amount of times you have to write return - and even reduce it onto one line.